lib/model/model.go

```diff
@@ -474,7 +441,7 @@ func (m *model) stopFolder(cfg config.FolderConfiguration, err error) {
 }
 
 // Need to hold lock on m.fmut when calling this.
-func (m *model) removeFolderLocked(cfg config.FolderConfiguration) {
+func (m *model) deleteFolderEntriesLocked(cfg config.FolderConfiguration) {
```

Comment:
Why the rename?

Reply:
> And I renamed `removeFolderLocked` to `deleteFolderEntriesLocked`, as it isn't the locked equivalent to `removeFolder`.

I consider the name misleading, given `removeFolder` is meant to actually completely remove the folder (as in it's no longer in config and we drop all we know from db), while this is used e.g. when pausing a folder as well.

Reply:
Stop? Cleanup? Shutdown?

Reply:
`cleanupFolderLocked` should work. Stop is taken and Shutdown not really accurate: It just removes the entries for said folder in all of the maps protected by fmut. That's why I thought a name representing that would be good. So maybe `deleteFolderMapEntriesLocked`? I am fine with either.